frontend/kesaseteli/employer/src/hooks/application/useLoadDraftOrCreateNewApplication.ts

```diff
@@ -0,0 +1,39 @@
+import useApplicationsQuery from 'kesaseteli/employer/hooks/backend/useApplicationsQuery';
```

Comment:
renamed and changed from [useLoadDraftOrCreateNewApplicationEffect](https://github.com/City-of-Helsinki/yjdh/pull/1135/files#diff-c4ff60cf532d7543ab024f4e155f3177e146f17a7eaf69ec93df87ae8ff8c2d5) because it's no more useEffect hook.